### DISCUSSION TITLE
test_server and test_client are deployed on different networks on docker

### DISCUSSION THREAD

**Discussion by LLL2000L** [alibaba/xquic#385] [Q&A]
Hello, I have a problem using the XQUIC library. test_server and test_client are deployed on docker. If the client and server are configured on different networks, I also added a similar NAT device to achieve network interoperability. However, the connection between the client and the server failed during runtime. The client can Ping. The server is connected, but the connection fails. If they are on the same network, they can be connected. What is the reason and how to solve it?

**Comment by Kulsk** [comment]
did you tried a well-known port? I think the Network link is OK while some mechanism between C/S is functional to make transfer fails, like the firewall or the router config.
